pkg/apis/monitoring/v1/types.go

```diff
@@ -1122,6 +1122,9 @@ type RemoteReadSpec struct {
 	TLSConfig *TLSConfig `json:"tlsConfig,omitempty"`
 	// Optional ProxyURL.
 	ProxyURL string `json:"proxyUrl,omitempty"`
+	// Optional FilterExternalLabels. This translates in filter_external_labels
+	// in prometheus.yaml configuration.
+	FilterExternalLabels bool `json:"filterExternalLabels,omitempty"`
```

Comment:
The default Prometheus value is `true` so we need a bool pointer (otherwise the flag would be automatically disabled for all users after upgrade).

```suggestion
	FilterExternalLabels *bool `json:"filterExternalLabels,omitempty"`
```